Utils/Dataflow/025_chicagoES/README

```diff
@@ -25,3 +25,10 @@ output):
 ---------
 Goes to stage 016.
 Contain same fields as input data, with additional fields.
+
+4. Samples
+----------
+The 'output' directory contains 2 samples:
+
+sample.ndjson - 2016 data (from 009's 2016 sample, 091 was skipped)
```

Comment:
This looks a bit distracting, maybe update the 2016 sample as well?

Reply:
See comment in #201.

Reply:
Per discussion in #201: the samples here will be updated. Done, others will follow.